src/Form/OgChangeMultipleRolesFormBase.php

```diff
@@ -93,6 +93,17 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#value' => $this->t('Submit'),
     ];
 
+    // Set selected memberships & group in form state. This can be used by
+    // developers to alter the roles form.
+    $memberships = $this->getMemberships();
+    if (empty($memberships)) {
+      return $form;
+    }
+    $membership = current($memberships);
+    $group = $membership->getGroup();
```

Comment:
Can this be removed? We can extract the group from the Membership

Reply:
Removed